src/main/webapp/app/pages/actionableGenesPage/ActionableGenesPage.tsx

```diff
@@ -540,6 +567,108 @@ export default class ActionableGenesPage extends React.Component<
   ];
 
   render() {
+    const levelSelectionSection = [];
+    for (const key in LEVEL_TYPES) {
+      if (LEVEL_TYPES[key]) {
+        levelSelectionSection.push(
+          // <LevelSelectionRow
+          //   levelType = {LEVEL_TYPES[key]}
+          //   collapseStatus = {this.collapseStatus}
```

Comment:
The change of `collapseStatus` will not re-render the `LevelSelectionRow`, but the `levelSelected` will. Any ideas to make the `collapseStatus` works as `levelSelected`?

Reply:
The change of `levelSelected` seems could re-render() `LevelSelectionRow`. What makes those two different?